### PR TITLE
LPD-34682 Integration test

### PR DESCRIPTION
**Pull request by ghost**
https://liferay.atlassian.net/browse/LPD-36223

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-payment-test/src/testIntegration/java/com/liferay/commerce/payment/service/test/CommercePaymentEntryLocalServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,682 @@
...
+		Assert.assertEquals(
+			CommercePaymentEntryConstants.STATUS_PENDING,
+			commercePaymentEntry.getPaymentStatus());
+		Assert.assertEquals(
+			CommercePaymentEntryConstants.TYPE_PAYMENT,
+			commercePaymentEntry.getType());
+	}
+
+	@Test
+	public void testAddOrUpdaterCommercePaymentEntry2() throws Exception {
```
Comment (dsitu) [inline modules/apps/commerce/commerce-payment-test/src/testIntegration/java/com/liferay/commerce/payment/service/test/CommercePaymentEntryLocalServiceTest.java:147]: Typo, should be 
`testAddOrUpdateCommercePaymentEntry2`

Comment (dsitu) [inline modules/apps/commerce/commerce-payment-test/src/testIntegration/java/com/liferay/commerce/payment/service/test/CommercePaymentEntryLocalServiceTest.java:147]: Typo, should be
`testAddOrUpdateCommercePaymentEntry2`

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-payment-test/src/testIntegration/java/com/liferay/commerce/payment/service/test/CommercePaymentEntryLocalServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,682 @@
...
+
+		CommerceTestUtil.updateBackOrderCPDefinitionInventory(
+			_cpInstance.getCPDefinition());
+
+		_commerceOrder = CommerceTestUtil.addB2CCommerceOrder(
+			_user.getUserId(), _commerceChannel.getGroupId(),
+			_commerceCurrency);
+
+		_commerceOrder = _commerceOrderLocalService.getCommerceOrder(
+			_commerceOrder.getCommerceOrderId());
```
Comment (dsitu) [inline modules/apps/commerce/commerce-payment-test/src/testIntegration/java/com/liferay/commerce/payment/service/test/CommercePaymentEntryLocalServiceTest.java:101]: Everything looks good, just curious why we need to do a getCommerceOrder here when it should be returned from above. Im assuming some changes happen in the backend after its returned above?

**Review by dsitu — APPROVED** [review]

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:forward

**Pull request closed without merging**